src/core/Akka.Remote/Transport/DotNetty/DotNettyTransport.cs

```diff
@@ -131,13 +131,17 @@ protected DotNettyTransport(ActorSystem system, Config config)
             System = system;
             Config = config;
 
+            // Helios compatibility
             if (system.Settings.Config.HasPath("akka.remote.helios.tcp"))
             {
-                var heliosFallbackConfig = system.Settings.Config.GetConfig("akka.remote.helios.tcp");
+                var heliosFallbackConfig = system.Settings.Config.GetConfig("akka.remote.helios.tcp")
+                    .WithFallback("transport-class = \"Akka.Remote.Transport.Helios.HeliosTcpTransport, Akka.Remote.Transport.Helios\"");
```

Comment:
Moved the hard coded HOCON settings from pidgeon.conf here, value only get added when necessary

Reply:
Do we still need to support Helios?